include/model.hpp

```diff
@@ -0,0 +1,61 @@
+/*
+** EPITECH PROJECT, 2022
+** indie
+** File description:
+** models
+*/
+
+#ifndef MODELH
+    #define MODELH
+
+    struct Model;
+    struct Vector3;
+    struct Color;
+
+    #include <memory>
+
+namespace indie {
+
+    /**struct Model;
+     * @brief Model 3D object
+     */  
+    class Model {
+        public:
+            /**
+             * @brief Construct a new model object
+             * @param filename path to the model file 
+             */
+            Model(const char *filename);
```

Comment:
We're in cpp, better use "const std::string &" than "const char *". You can use ".c_str()" after